### PR TITLE
added JourneyRelations

### PR DESCRIPTION
**Pull request by ue71603**
Adding JourneyRelations to TripInfoResult

If we need it (was used by SBB Online trip planner www.sbb.ch to show the relations).

### CODE REVIEW

**Review by skinkie — APPROVED** [review]

**Review by trurlurl — APPROVED** [review]

**Comment by skinkie** [comment]
@ue71603 merge or wait for a _german_ and/or _french_ approval?

**Pull request merged**
